### PR TITLE
Add support for redis `execute`

### PR DESCRIPTION
**Pull request by karthik2804**
This PR adds support for the redis `execute` method to allow executing of arbitrary redis commands.

### CODE REVIEW

File: crates/spin-js-engine/src/lib.rs [hunk 1]
```diff
@@ -329,6 +330,74 @@ fn math_rand(context: &Context, _this: &Value, _args: &[Value]) -> Result<Value>
     context.value_from_f64(thread_rng().gen_range(0.0_f64..1.0))
 }
 
+fn redis_exec(context: &Context, _this: &Value, args: &[Value]) -> Result<Value> {
+    enum Parameter {
+        Int64(i64),
+        Binary(ByteBuf),
+    }
+
+    println!("execcuting exec");
```
Comment (dicej) [inline crates/spin-js-engine/src/lib.rs]: Can we remove this?

**Review by dicej — APPROVED** [review]

File: crates/spin-js-engine/src/lib.rs [hunk 2]
```diff
@@ -329,6 +330,74 @@ fn math_rand(context: &Context, _this: &Value, _args: &[Value]) -> Result<Value>
...
+                        let mut serializer = Serializer::from_context(context)?;
+                        val.serialize(&mut serializer)?;
+                        arr.append_property(serializer.value)?;
+                    }
+                }
+            }
+            return Ok(arr);
+        }
+        _ => bail!(
+            "expected a two arguments (address, key), got {} arguments",
```
Comment (dicej) [inline crates/spin-js-engine/src/lib.rs]: This message doesn't match the actual signature.

Comment (karthik2804) [reply crates/spin-js-engine/src/lib.rs]: Thanks for catching that.

File: crates/spin-js-engine/src/lib.rs (hunk 1 above)
Comment (karthik2804) [reply crates/spin-js-engine/src/lib.rs]: done.

**Pull request merged**